Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -287,7 +287,15 @@ static NSString *const STPSDKVersion = @"16.0.6";
  
  The default value is NO.
  */
-@property (class, nonatomic, getter=isJCBPaymentNetworkSupported) BOOL JCBPaymentNetworkSupported;
+@property (class, nonatomic, getter=isJCBPaymentNetworkSupported) BOOL JCBPaymentNetworkSupported __attribute__((deprecated("Set additionalApplePayNetworks = @[PKPaymentNetworkJCB] instead")));
+
+/**
+ Set this to enable additional networks for Apple Pay if needed.
+ By default, the SDK supports Amex, MasterCard, Visa, and Discover.
+ 
+ For example, `Stripe.additionalSupportedApplePayNetworks = @[PKPaymentNetworkJCB];` enables JCB (note this requires onboarding from JCB and Stripe).
+ */
+@property (class, nonatomic, nonnull) NSArray<PKPaymentNetwork> *additionalSupportedApplePayNetworks;
```

Comment:
I'm worried that having `supportedPKPaymentNetworks ` and `additionalSupportedApplePayNetworks` will be confusing. What if we just make `supportedPKPaymentNetworks` writeable?

Reply:
I'm actually not a huge fan of `supported` in this context. It seems to imply what Stripe supports, but we're allowing arbitrary editing?

Reply:
Yeah I thought the same (even wrote out a `setSupportedPKPaymentNetworks`).  I decided against it b/c:
1. `supportedPKPaymentNetworks` isn't public. 
2. We don't want to let users remove the defaults (mastercard, etc.).  `setSupportedPKPaymentNetworks` implies arbitrary editing, but we just want to let them *add* to the defaults.  

Would rewording to `accept` be any better?  Maybe something like "The SDK accepts Amex, MasterCard, Visa, and Discover for Apple Pay.  Set this property to enable other card networks in addition to these." is clearer.

Reply:
Makes sense. Maybe `enabled` instead of `supported` or `accept`? Defintely + on that docstring